check/greynoise.go

```diff
@@ -0,0 +1,67 @@
+package check
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"strings"
+)
+
+var greynoiseurl = "https://api.greynoise.io/v3/community/"
+
+type grey struct {
+	IP             string `json:"ip"`
+	Noise          bool   `json:"noise"`
+	Riot           bool   `json:"riot"`
+	Classification string `json:"classification"`
+	Name           string `json:"name"`
+	Link           string `json:"link"`
+	LastSeen       string `json:"last_seen"`
+	Message        string `json:"message"`
+}
+
+// Json implements IpInfo.
+func (g grey) Json() ([]byte, error) {
+	return json.Marshal(g)
+}
+
+// Summary implements IpInfo.
+func (g grey) Summary() string {
+	return fmt.Sprintf("IP %s, riot: %v, message: %s", g.IP, g.Riot, na(g.Message))

```

Comment:
IP address is redundant in the summary output:

```
--- 91.228.166.47 ---
db-ip.com       Petržalka, Slovakia
dns name        skh1-webredir01-v.eset.com
greynoise.io    IP 91.228.166.47, riot: false, message: IP not observed scanning the internet or contained in RIOT data set.
iptoasn.com     ESET-AS
is on AWS       false
isc.sans.edu    attacks: 0, abuse contact: domains@eset.sk
ping            0% packet loss (5/5), avg round-trip 5 ms
tls             TLS 1.3, exp. 2024/01/02!!, www.eset.com, eset.com
virustotal.com  network: 91.228.164.0/22, SAN: www.eset.com, eset.com
malicious prob. 15% (2/13) 🤏
```

Also, what does it mean that "IP is not contained in RIOT data set"?